azure-quantum/azure/quantum/optimization/streaming_problem.py

```diff
@@ -148,11 +199,43 @@ def add_terms(self, terms: List[Term]):
                     "Cannot add terms after problem has been uploaded"
                 )
 
-            term_couplings = [len(term.ids) for term in terms]
-            max_coupling = max(term_couplings)
-            min_coupling = min(term_couplings)
-            self.__n_couplers += sum(term_couplings)
-            self.stats["num_terms"] += len(terms)
+            for term in terms:
+                if isinstance(term, Term):
+                    n = len(term.ids)
+                    max_coupling = max(max_coupling, n)
```

Comment:
This is throwing an error in the live tests because `max_coupling` is not defined

Reply:
I was able to fix this locally by adding the following lines above ln 202:
```python
term_couplings = [len(term.ids) for term in terms]
max_coupling = max(term_couplings)
min_coupling = min(term_couplings)
```